ofl/notosanssignwriting/METADATA.pb

```diff
@@ -0,0 +1,22 @@
+name: "Noto Sans SignWriting"
```

Comment:
Are we sure about SignWriting? not Sign Writing, and not Signwriting. If yes, then it should be added to the camel case exception list: https://github.com/googlefonts/fontbakery/blob/main/Lib/fontbakery/data/googlefonts/camelcased_familyname_exceptions.txt
cc @davelab6 @chrissimpkins

Reply:
The official name of the script is, indeed, SignWriting.

Reply:
CamelCase accepted, the name should be added to the exception list in fontbakery